wgsl/index.bs

```diff
@@ -2830,6 +2830,8 @@ An expression [=shader-creation error|must not=] evaluate to a runtime-sized arr
 
 The element count expression |N| of a fixed-size array is subject to the following constraints:
 * It [=shader-creation error|must=] be an [=override-expression=].
+  * Note:  To qualify for type-equivalency, the override expression must be an override identifier.
```

Comment:
The content of the note is fine for me.
But it needs to be indented more.  Indentation is by multiples of 4.

```
    $ bikeshed --die-on=warning spec "wgsl/index.bs" "wgsl/index.bs.built.html" 
      FATAL ERROR: Line 2834 isn't indented enough (needs 1 indent) to be valid Markdown:
      "See "Workgroup variables sized by overridable constants"."
       ✘  Did not generate, due to fatal errors
    Command `bikeshed --die-on=warning spec "wgsl/index.bs" "wgsl/index.bs.built.html" ` failed with exit code: 2.
```